server/lobbyconnection.py

```diff
@@ -362,20 +362,17 @@ async def command_admin(self, message):
             if await self.player_service.has_permission_role(
                 self.player, "ADMIN_BROADCAST_MESSAGE"
             ):
-                tasks = []
                 for player in self.player_service:
                     # Check if object still exists:
                     # https://docs.python.org/3/library/weakref.html#weak-reference-objects
                     if player.lobby_connection is not None:
-                        tasks.append(
-                            player.lobby_connection.send_warning(message_text)
-                        )
+                        with contextlib.suppress(DisconnectedError):
```

Comment:
Either this is needed in which case it should be used in both places, or it's not needed in which case it shouldn't be used in either place